srcpkgs/shortwave/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'shortwave'
+pkgname=shortwave
+version=2.0.1
+revision=1
+_short_rev="9bfc5ee"
+build_style=meson
+build_helper=rust
+configure_args=" -Dtag=${_short_rev}"
+hostmakedepends="cargo gettext glib-devel pkg-config rust sqlite-devel"
+makedepends="gstreamer1-devel gst-plugins-bad1-devel gtk4-devel libadwaita-devel openssl-devel rust-std sqlite-devel"
+depends="gst-plugins-good1"
+checkdepends="desktop-file-utils"
+short_desc="GTK internet radio player, written in Rust"
+maintainer="Philipp David <pd@3b.pm>"
+license="GPL-3.0-or-later"
+homepage="https://gitlab.gnome.org/World/Shortwave"
+distfiles="https://gitlab.gnome.org/World/Shortwave/uploads/1c5adbe8c3181b2ba81d944a8f9a889d/${pkgname}-${version}.tar.xz"
```

Comment:
Can we use the usual URL? I think this URL will break on the next update and will have to be fixed.

Reply:
The .tar.xz includes all dependencies, which I probably used as a workaround to some git pulls during build failing because of deleted branches and the like.

It also sounded nice not having to hit the Internet after fetching the tarball, but that was never the case since some crates were brokenly outdated from the start.

Reply:
Interesting, I still have to download a lot: 
```=> shortwave-2.0.1_1: running do_build ...
ninja: Entering directory `build'
[1/4] Generating de.haeckerfelix.Shortwave.desktop_data_merge with a custom command
[2/4] Generating de.haeckerfelix.Shortwave.metainfo.xml_data_merge with a custom command
[3/4] Generating de.haeckerfelix.Shortwave_gresource with a custom command
[3/4] Generating cargo-build with a custom command
** RUST VERSION **
rustc 1.53.0
** RELEASE MODE **
    Updating git repository `https://gitlab.gnome.org/bilelmoussaoui/libadwaita-rs`
    Updating git submodule `https://github.com/gtk-rs/gir.git`
    Updating git submodule `https://github.com/gtk-rs/gir-files.git`
    Updating crates.io index
    Updating git repository `https://gitlab.freedesktop.org/gstreamer/gstreamer-rs`
    Updating git submodule `https://github.com/gtk-rs/gir`
    Updating git submodule `https://github.com/gtk-rs/gir-files`
    Updating git submodule `https://gitlab.freedesktop.org/gstreamer/gir-files-rs.git`
    Updating git repository `https://github.com/gtk-rs/gtk4-rs`
    Updating git submodule `https://github.com/gtk-rs/gir.git`
    Updating git submodule `https://github.com/gtk-rs/gir-files.git`
    Updating git repository `https://github.com/gtk-rs/gtk-rs`
    Updating git submodule `https://github.com/gtk-rs/gir`
    Updating git submodule `https://github.com/gtk-rs/gir-files`
 Downloading crates ...
  Downloaded addr2line v0.14.1
  Downloaded data-encoding v2.3.2
  Downloaded cfg-expr v0.7.4
...
```
I think we can leave it like this for now, we can always change it.